conf/janus.plugin.audiobridge.jcfg.sample

```diff
@@ -9,6 +9,7 @@
 # audiolevel_event = true|false (whether to emit event to other users or not, default=false)
 # audio_active_packets = 100 (number of packets with audio level, default=100, 2 seconds)
 # audio_level_average = 25 (average value of audio level, 127=muted, 0='too loud', default=25)
+# default_prebuffering = number of packets to buffer before decoding each particiant (default=6)
```

Comment:
typo in word "partici**p**ant" here and in plugins/janus_audiobridge.c

Reply:
Thanks, I'll fix this :+1: